src/notification/examples/notification-drawer.js

```diff
@@ -114,8 +115,11 @@
    </div>
    <span ng-if="notification.status" class="{{'pull-left ' + $ctrl.customScope.getNotficationStatusIconClass(notification)}}" ng-click="$ctrl.customScope.markRead(notification)"></span>
    <div class="drawer-pf-notification-content" ng-click="$ctrl.customScope.markRead(notification)">
-     <span class="drawer-pf-notification-message" tooltip-append-to-body="true" tooltip-popup-delay="500" tooltip-placement="bottom" tooltip="{{notification.message}}">
-      {{notification.message}}
+     <span ng-if="!$ctrl.htmlContent" class="drawer-pf-notification-message" tooltip-append-to-body="true" tooltip-popup-delay="500" tooltip-placement="bottom" tooltip="{{notification.message}}">
```

Comment:
My only concern is that we put the entire notification message in the tooltip and I don't know how/if it works when there is HTML in there.  I can't get tooltips to appear for the pfNotificationDrawer ngDoc example?